olp-cpp-sdk-core/src/http/android/NetworkAndroid.cpp

```diff
@@ -617,52 +640,48 @@ jobjectArray NetworkAndroid::CreateExtraHeaders(
   }
 
   // Merge vector of pair strings into one JNI array
-  const size_t header_count = extra_headers.size() * 2;
-  jobjectArray headers = (jobjectArray)env->NewObjectArray(
+  const size_t extra_headers_count = 1;
```

Comment:
do you need and extra variable for this? Why not:
```
const size_t header_count = (headers.size() + 1) * 2;
```

Reply:
Not valid anymore